native_client/wasm/package.json.in

```diff
@@ -0,0 +1,14 @@
+{
+  "name" : "$(PROJECT_NAME)",
```

Comment:
Did you test the packaged library to see if it's possible to use it from within an NPM-based project?

Reply:
I did test it with an example I would like to contribute eventually to `stt-examples` but I think it is a good idea to test it in the CI pipeline as well. I'll add a test step for it.